src/main/java/Duke.java

```diff
@@ -1,10 +1,85 @@
+import java.util.Scanner;
+
 public class Duke {
+    static WordList wordList;
+
     public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        wordList = new WordList();
+
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+
+        replyWelcomeMessage();
+        String input;
+        while (true) {
+            try {
+                input = sc.nextLine();
+                checkEmpty(input);
+                Object[] parseResult = InputParser.parseInput(input);
+                InputType inputType = (InputType) parseResult[0];
+                String[] value = (String[]) parseResult[1];
```

Comment:
I think parseResult and value could both be renamed in plural form (results and values perhaps?) to be consistent with naming conventions. I noticed this in a few other places.